components/openvsx-proxy/pkg/modifyresponse.go

```diff
@@ -93,13 +94,42 @@ func (o *OpenVSXProxy) ModifyResponse(r *http.Response) error {
 	}
 
 	// no error (status code < 500)
+	body := rawBody
 	contentType := r.Header.Get("Content-Type")
 	if strings.HasPrefix(contentType, "application/json") {
+		compressedResponse := strings.EqualFold(r.Header.Get("Content-Encoding"), "gzip")
```

Comment:
for me, it'd be easier to understand that this is a bool, if we name this: `isCompressedResponse`